cmd/taskctl/taskctl_test.go

```diff
@@ -16,15 +16,21 @@ type cmdRunTestInput struct {
 	errored     bool
 	exactOutput string
 	output      []string
+	ctx         context.Context
 }
 
 func cmdRunTestHelper(t *testing.T, testInput *cmdRunTestInput) {
 	t.Helper()
+	ctx := context.TODO()
```

Comment:
Hmmm is `TODO` just a state of the Context?

Reply:
yes for tests it's a TODO context with nothing in it